.github/ISSUE_TEMPLATE/config.yml 

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
BWT, this file had invalid name - some garbage characters at the end.

Reply:
I totally know where it came from! I should have checked it... 
When I saved it, I thought it was space and Kate will trim it. 
This damn documentation... Hahaha.

Thank you!

Reply:
It is funny, but the problem it produced made GIT repository uncloneable for Linux and Windows!!!

My Linux machine simply refused to clone/pull from repo from the same error: https://github.com/martinrotter/rssguard/runs/3333029017?check_suite_focus=true#step:2:520

Reply:
> the problem it produced made GIT repository uncloneable for Linux and Windows (...)

Say what! 

I am on ext4, obviously.

I looked into it, and it _is_ just a space! 
I, then, went ahead and tested such thing again, and it does _not_ get trimmed but acceptable perfectly.

Ext4 sure can be scary at times...

Reply:
It probably wasn't regular ASCII space, maybe it was non-breaking space or some other exotic space/tab? Anyway, just could not even "touch" the file, probably limitation of NTFS.

Reply:
It is mere space!
I was surprised myself to find that, but it is just a space.

The only thing I find strange is that when I copy from the filename only that space alone, it is not being copied as regular space, however when I copy it together with some letters, it is nothing but a space.
Well, I use parcellite as clipboard manager, so it may be to blame with this one, but I have also typed the tailing space by my keyboard, and result is the same.

I dare say it is an issue of some of KDE components of Kate and Kwrite, because when I did the same in Dolphin (file manager, also by KDE), the space was duly trimmed.

Reply:
As for KDE: I am long-time user of MATE. It just does what I want it to. It is simple, very lightweight and cozy. I used KDE many years ago (some 4.X version) and wanted to test it back and I mean - the visual expecience was immense but usability and bugs - meh. For example I was not for some crazy bug able to assing keyboard shortcut to launch "Tilis" (drop-down terminal emulator) - it just did not launch no matter what. That one bug was stopper for me. :D I guess I will try once more in a year.

Reply:
Well, I am myself after half a year moved to Xfce (the very first DE after which I understand that Linux may be not so bad), however decided on KDE, the reason were Dolphin and Kate, and I doubt that I will ever let go of them.
Not of Kate, anyhow.

Reply:
Try Double Commander - serious shit, trust me.

As for Kate - well, I now use "micro" when in terminal and "vscode" when in GUI. I know, i know.

Reply:
At least not Vi, nor Vim, huh.

Nano is my terminal choice, though I would still like to use line numbers and be able to use copying by mouse normally.

Reply:
Try micro, it has mouse support.

Reply:
It has no problems with mouse support, just the line numbers are being copied together with text.

Reply:
I will try that, of course.

Reply:
In "micro" when I drag mouse selection, then line numbers are not selected.

![image](https://user-images.githubusercontent.com/1255302/129523819-059b036c-d0ca-410e-8d14-28284662f24e.png)

Reply:
I already love it! :heart:

Reply:
Tilix: found nothing too appealing for me to move from `xfce4-terminal`, which also has drop-down support. GTK is still as sluggish, when scrolling through huge documents.
Double Commander: too serious for someone such as myself.
Micro: Perhaps worth it. It will take some time to get used to "ordinary" shortcuts, but the reason is in the screenshot above.
VSCode: ...Didn't try it. It will likely be as bad as the Atom.